src/catalog/src/durable/initialize.rs

```diff
@@ -105,6 +108,8 @@ pub(crate) async fn initialize(
                 MZ_INTERNAL_SCHEMA_ID,
                 INFORMATION_SCHEMA_ID,
                 MZ_UNSAFE_SCHEMA_ID,
+                MZ_CATALOG_UNSTABLE_SCHEMA_ID,
+                MZ_INTROSPECTION_SCHEMA_ID,
             ])
```

Comment:
Adding system schemas here seems unnecessary, given that `SCHEMA_ID_ALLOC_KEY` is only used for user schemas, which have a different ID namespace (`SchemaId::User` instead of `SchemaId::System`). But I didn't want to change the way we do things in this PR.